cfme/utils/release.py

```diff
@@ -49,6 +49,7 @@ def clean_commit(commit_msg):
     "RHV",
     "sprout",
     "tech-debt",
+    "test-automation",
```

Comment:
How is this different from new-test-or-feature? what is the use case?

Reply:
It is for PR's that are automating manual test cases, so that we can collect metrics about number of manual cases automated per sprint.